collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -612,6 +611,11 @@ protected void onActivityResult(int requestCode, int resultCode,
             return;
         }
 
+        if (intent == null) {
+            Timber.i("The intent has a null value for requestCode: " + requestCode);
```

Comment:
I think we should switch to Crashlytics https://github.com/opendatakit/collect/issues/1040 maybe it looks better there.